README.md

```diff
@@ -0,0 +1,3 @@
+# 2021-jan-pairs-game
+#Access the Live Version: https://dev.io-academy.uk/projects/2021-jan/2021-jan-pairs-game/
```

Comment:
delete the hash at the start of this line and the one on the line below

Reply:
Have done!